packages/code-studio/src/dashboard/DashboardContainer.jsx

```diff
@@ -56,29 +57,6 @@ const log = Log.module('DashboardContainer');
 const RESIZE_THROTTLE = 100;
 
 export class DashboardContainer extends Component {
-  static dehydratePanelConfig(config) {
```

Comment:
I think I did something similar in my separating dashboards branch (moving dehydratePanelConfig to LayoutUtils)